scripts/src/main/resources/scripts/command/vscode

```diff
@@ -166,8 +166,7 @@ do
     doCreateIdeScript vscode
   elif [ "${1}" = "add-plugin" ]
   then
-    shift
-    doAddPlugin "${@}"
+    doAddPlugin "--install-extension ${2}"
```

Comment:
shouldn't the `--install-extension` option be added inside `doAddPlugin` function?

Reply:
There are 2 Functions the doAddPlugins and doAddPlugin

The doAddPlugins would break if I add the --install-extension into the doAddPlugin Function

because the doAddPlugins Functions passes a String with multiple plugins to the doAddPlugin.
Multiple plugins have to be installed with the following format : --install-extension plugin1 --install-extension plugin 2 ...
If I add the "--install-extension" into the doAddPlugin function it would call vscode with --install-extension --install-extension which would lead to an error.

Reply:
Refactored to support multiple plugin-ids as parameter.